lib/features/pokemon_details/widgets/stats_tab_view.dart

```diff
@@ -1,19 +1,52 @@
 import 'package:flutter/material.dart';
-import 'package:pokedex/model/dto/pokemon_dto.dart';
+import 'package:pokedex/api/pokedex_api_spec/model/stats.dart';
+import 'package:pokedex/widgets/app_text_widgets.dart';
+import 'package:pokedex/widgets/spacings.dart';
 
 class StatsTabView extends StatelessWidget {
   const StatsTabView({
     super.key,
-    required this.details,
+    required this.stats,
   });
 
-  final PokemonDto details;
+  final List<Stats> stats;
 
   @override
   Widget build(BuildContext context) {
     /// TODO: This will be improved once loading is implemented
-    final stats = details.stats.isEmpty ? '' : details.stats[0].stats!.name;
 
-    return Center(child: Text(stats));
+    return ListView.builder(
+      padding: const EdgeInsets.all(20.0),
+      itemCount: stats.length,
+      itemBuilder: (context, index) {
+        return Column(
+          children: [
+            Row(
+              children: [
+                SizedBox(
+                  width: MediaQuery.of(context).size.width - 280.0,
+                  child: DetailText(text: stats[index].stats!.name),
```

Comment:
i think you can assume already what I'm going to comment here? 😅 

kindly apply as well on other affected parts here 😉